test/AccountManager.test.ts

```diff
@@ -99,38 +99,37 @@ describe('AccountManager', function () {
     });
 
     describe('removeAccount()', function () {
-      let fakeAccounts: Array<Wallet>;
+      let randomAccounts: Array<Wallet>;
+      let initialLength: number;
 
       beforeEach(function () {
-        fakeAccounts = new Array<Wallet>(3).fill(Wallet.createRandom());
+        initialLength = 3;
+
+        randomAccounts = new Array<Wallet>(initialLength).fill(
+          Wallet.createRandom()
+        );
```

Comment:
I've just realised that this creates an array filled with the same object. if you check the address of the wallets created, they're all the same

Reply:
Fixed.